src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -213,3 +213,17 @@ anoremenu <script> Gui.Treeview.Toggle :call <SID>TreeViewShowToggle()
 function GuiShowContextMenu() range
 	call rpcnotify(0, 'Gui', 'ShowContextMenu')
 endfunction
+
+" Notify Gui of any cursor movement, used by GuiScrollBar
+function GuiCursorMovedUpdateScrollBar()
+	call rpcnotify(0, 'Gui', 'CursorMovedUpdateScrollBar', line('w0'), line('$'), winheight(winnr()))
+endfunction
+
+autocmd CursorMoved,CursorMovedI,VimResized * call GuiCursorMovedUpdateScrollBar()
+
+" Show/Hide the Gui ScrollBar
+function! s:GuiScrollBar(enable) abort
+	call rpcnotify(0, 'Gui', 'SetScrollBarVisible', a:enable)
+endfunction
+
+command! -nargs=1 GuiScrollBar call s:GuiScrollBar(<args>)
```

Comment:
This would be a call in ginit.vim right?

Update the docs with GuiScrollBar

Reply:
Yes, this would be off-by-default and enabled from ginit.vim